CMakeLists.txt

```diff
@@ -59,6 +61,16 @@ set(UMF_PROXY_LIB_BASED_ON_POOL
 set_property(CACHE UMF_PROXY_LIB_BASED_ON_POOL
              PROPERTY STRINGS ${KNOWN_PROXY_LIB_POOLS})
 
+if(UMF_BUILD_TESTS
+   AND DEFINED ENV{CI}
```

Comment:
We can verify if `"$CI" == "true"` exactly.

Reply:
tbh, I couldn't make it work - I'm not sure how CMake is treating env vars...

anyway, this shouldn't be a big problem since you'd have to have exactly `CI` variable defined in your OS, for this to fail, and even so, it's just a matter of adding extra param to the CMake command.

Reply:
It is OK.